src/main/webapp/app/shared/components/button.component.ts

```diff
@@ -24,7 +25,13 @@ export enum ButtonSize {
 @Component({
     selector: 'jhi-button',
     template: `
-        <button [ngClass]="['jhi-btn', 'btn', btnType, btnSize]" ngbTooltip="{{ tooltip | translate }}" [disabled]="disabled || isLoading" (click)="onClick.emit($event)">
+        <button
+            style="width: 100%"
+            [ngClass]="['jhi-btn', 'btn', btnType, btnSize]"
+            ngbTooltip="{{ tooltip | translate }}"
+            [disabled]="disabled || isLoading"
+            (click)="routerLink ? navigate() : onClick.emit($event)"
```

Comment:
I think this would be a good case for a Angular Directive, because we should enable the CTRL+Click behavior for links (open in new tab: https://github.com/ls1intum/Artemis/issues/611).